style.css

```diff
@@ -661,6 +661,16 @@ ul {
 
 .logo img {
   max-height: 37px;
+  vertical-align: middle;
+}
+
+.logo span {
+  margin: 0 10px 0 10px;
+  color: $brand_color;
+}
+
+.logo a:hover {
+  text-decoration: none;
```

Comment:
We should also target `:focus` and `:active` states as they will also show an underline.

i.e.: on focus (user navigates with the keyboard - keeps pressing TAB):

<img width="153" alt="Screenshot 2021-01-11 at 13 37 13" src="https://user-images.githubusercontent.com/1172767/104189097-1eee2480-5412-11eb-88fe-0ac230504330.png">

I also suggest updating the `.logo a` to use `display: inline-block;` so the focus outline becomes a rectangle instead:

<img width="145" alt="Screenshot 2021-01-11 at 13 40 03" src="https://user-images.githubusercontent.com/1172767/104189399-83a97f00-5412-11eb-8e67-bba74a883145.png">